dev/bots/test.dart

```diff
@@ -674,7 +674,8 @@ Future<void> _verifyVersion(String filename) async {
     exit(1);
   }
   final RegExp pattern = RegExp(r'^[0-9]+\.[0-9]+\.[0-9]+(-pre\.[0-9]+)?$');
```

Comment:
Doesn't this also need the `v` at the beginning? Or should the other one not have the `v`?
```suggestion
  final RegExp pattern = RegExp(r'^\d+\.\d+\.\d+(-pre\.\d+)?$');
```